core/tests/test_utils.py

```diff
@@ -830,9 +971,18 @@ def __init__(self, f):
 
     @property
     def times_called(self):
+        """Property that returns the number of times a function is called.
```

Comment:
a function --> the wrapped function
is called --> has been called

Missing Returns docstring?

Reply:
Done!